car/Musthardzik/index.ts

```diff
@@ -0,0 +1,63 @@
+class Car {
+
+	constructor(
+  private brand: string,
+  private model: string,
+  private version: string,
+  private year: number,
+  private isRunning: boolean,
+  private fuelLevel: number
+  ) {}
+
+  startCar() {
+    if (this.fuelLevel > 1 && this.isRunning === false) {
```

Comment:
what if fuelLevel will be 1? From logical point of view, car should start working

Reply:
i mean about `this.fuelLevel > 1`